gnovm/stdlibs/encoding/base64/base64.gno

```diff
@@ -496,7 +496,7 @@ func (enc *Encoding) Decode(dst, src []byte) (n int, err error) {
 	_ = enc.decodeMap
 
 	si := 0
-	for strconv.IntSize >= 64 && len(src)-si >= 8 && len(dst)-n >= 8 {
+	for len(src)-si >= 8 && len(dst)-n >= 8 {
```

Comment:
?
any modifications in stdlibs should be well documented with XXX. 
I would comment it out like /*strconv.IntSize >= 64 && */ above.